Logger/LogFactory.cs

```diff
@@ -1,10 +1,35 @@
-﻿namespace Logger;
-
-public class LogFactory
-{
-    public BaseLogger CreateLogger(string className)
-    {
-
-        return null;
-    }
-}
+﻿namespace Logger;
+#nullable enable
+
+public class LogFactory
+{
+    //create nullable path string
+    private string? path;   
+
+    //Configure given filePath as path
+    public void ConfigureFileLogger(string filePath)

```

Comment:
This method should be used in the createLogger method
```suggestion
    public void ConfigureFileLogger(string filePath)

```

Reply:
Thanks for pointing that out should be fixed now.